docs/source/gui.rst

```diff
@@ -1,4 +1,55 @@
 GUI User Manual
 ===============
 
-WIP...
+The FLFM GUI is a simple Python/Dash application to view and reconstruct FLFM images.
+
+For instructions on how to start the application see :ref:`usage_gui`.
+
+The following landing page will be shown upon opening the app in your browser.
+
+.. image:: ../_static/landing_page.png
+
+From here the three main tabs: ``PSF``, ``Light-Field-Image``, and ``3D Reconstruction`` are present. Clicking on any of
+these tabs will display their corresponding view. Both the ``PSF`` and ``Light-Field-Image`` tabs are used to load
+the relevant files to be reconstructed.
+
+Image files can be loaded by either selecting or by dragging and dropping them to the load section of the page. Upon
+doing so the image will be displayed, e.g.:
+
+.. image:: ../_static/loaded_psf.png
+
+If the PSF has not already been normalized, click on the ``Normalize PSF`` button to do so.
+
+.. note:: The PSF **must** be normalized for the final 3D reconstruction to be numerically accurate.
+
+.. note:: The ``Normalize PSF`` feature can only be used once per uploaded image to guard against accidentally normalizing more than once.
```

Comment:
Is this something we need to guard against? Normalizing again would leave the numbers unchanged right?